core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -494,6 +504,16 @@ public GenericContainer withCommand(String... commandParts) {
         return this;
     }
 
+    /**
+     * Add an extra host entry to be passed to the container
+     *
+     * @param hostEntry for e.g. "somehost:192.168.47.11"
+     * @return this
+     */
+    public GenericContainer withExtraHost(String hostEntry) {
```

Comment:
Please could we have this as two parameters, e.g. `withExtraHost(String hostname, String ipAddress)`?

I know people can refer to the Javadocs to see the "host:ip" format pattern, but I think having this directly apparent from the parameter names is a bit easier.